tasks/RedundantRBSSelectorTask.cpp

```diff
@@ -193,7 +202,24 @@ void RedundantRBSSelectorTask::updateHook()
     else if (secondary_is_new_and_valid) {
         _rbs_out.write(secondary_rbs);
     }
+    PoseDeviation pose_deviation;
+    pose_deviation.different = arePosesIncompatible(main_rbs, secondary_rbs);
+    pose_deviation.time = base::Time::now();
+    _pose_deviation.write(pose_deviation);
 }
+
+bool RedundantRBSSelectorTask::arePosesIncompatible(RigidBodyState const& first,
+    RigidBodyState const& second)
+{
+    base::Position first_position = first.getPose().position;
+    base::Position second_position = second.getPose().position;
+    bool position_status = (first_position - second_position).norm() > m_position_threshold;
+    bool roll_status = isDifferenceOutsideThreshold(first.getRoll(), second.getRoll(), m_roll_threshold);
```

Comment:
Imagine a threshold of 30 degrees, the first equal to 170 degrees and the second -170. This difference should be inside the threshold, but in the actual implementation it will be outside -> 170 -(-170) = 340